src/sentry/static/sentry/app/utils/dates.tsx

```diff
@@ -212,3 +212,8 @@ export function parsePeriodToHours(str: string): number {
       return -1;
   }
 }
+
+export const use24Hours = (): boolean => {
```

Comment:
IMO you don't need to explicitly type the return, since it can be inferred.

Reply:
Gotcha! This is my first time using TypeScript so opinions especially welcome :)